src/tests/verified_chats.rs

```diff
@@ -898,7 +898,7 @@ async fn test_no_unencrypted_name_if_verified() -> Result<()> {
         let chat_id = bob.create_chat(&alice).await.id;
         let msg = &bob.send_text(chat_id, "hi").await;
 
-        assert_eq!(msg.payload.contains("Bob Smith"), !verified);
+        assert_eq!(msg.payload.contains("Bob Smith"), false);
```

Comment:
So the behaviour for unverified chats is changed, now there is no unencrypted author's name. Previously it was added for compatibility with Thunderbird which doesn't use names from the encrypted part. But i'm ok with the change, this way we don't reveal our name to the network which is maybe more important